templates/profile.php

```diff
@@ -75,7 +76,8 @@ function pmpromd_the_title($title, $post_id = NULL)
 				{
 					global $wpdb;
 					$user_nicename = $_REQUEST['pu'];
-					$user = $wpdb->get_row("SELECT * FROM $wpdb->users WHERE user_nicename = '" . esc_sql($user_nicename) . "' LIMIT 1");
+					$user_identifier = apply_filters( 'pmpromd_user_identifier', 'user_nicename' );
+					$user = $wpdb->get_row("SELECT * FROM $wpdb->users WHERE `$user_identifier` = '" . esc_sql($user_nicename) . "' LIMIT 1");

```

Comment:
`pmpromd_profile_shortcode()` has this right, let's replace this direct DB query and use `get_user_by()` instead.

There's is at least one more query like this, please replace that too to ensure we have a completely normalized way to do the lookups now that we are opening up to other columns.